sdcm/ycsb_thread.py

```diff
@@ -161,9 +173,9 @@ def copy_template(self, docker):
                 docker.send_files(tmp_file.name, os.path.join('/tmp', 'dynamodb.properties'))
 
             with tempfile.NamedTemporaryFile(mode='w+', encoding='utf-8') as tmp_file:
-                tmp_file.write(aws_empty_file)
+                tmp_file.write(aws_credentials_file)
```

Comment:
Yes, I wondered why this (definitely non-empty) file was called "aws_empty_file" :-)

Reply:
yeah, that was fine for alternator sake. (I've started with creating empty file (just to find out it can't be completely empty, you need the fields and they can be empty)